crates/joining_code/Cargo.toml

```diff
@@ -9,7 +9,7 @@ name = "hc_joining_code"
 crate-type = [ "cdylib", "rlib" ]
 
 [dependencies]
-hdk = "0"
+hdk = "=0.0.107"
 serde = "1.0"
 derive_more = "0.99"
-holo_hash = "0"
+holo_hash = { version = "*", features = ["encoding"]}
```

Comment:
```suggestion
holo_hash = { version = "=0.0.7", features = ["encoding"]}
```

Reply:
because it will do that only if zome/chat is the latest released version released, if not it will default to the newer release. This is why we end up updating hc-utils crate all the time.